X-AIP.md

```diff
@@ -29,17 +25,13 @@ The motivation is critical for AIPs that want to change Aave. It should clearly
 
 The technical specification should describe the syntax and semantics of any new feature.
 
-## Rationale
-
-The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
-
-## Test Cases
-
-Test cases for an implementation are mandatory for AIPs but can be included with the implementation.
+## References
```

Comment:
i think instead of having #test cases and #implementation it makes sense to just add a #references section where ppl can add what they want.